renovate.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended"],
+  "dependencyDashboard": true,
+  "pinVersions": true,
+  "vulnerabilityAlerts": {
+    "groupName": "security updates",
+    "enabled": true,
+    "schedule": [],
+    "labels": ["security"],
+    "prCreation": "immediate"
+  },
+  "osvVulnerabilityAlerts": true,
+  "transitiveRemediation": true,
+  "baseBranches": ["main"],
+  "packageRules": [
```

Comment:
what happens to minor updates? -> is there a specific reason for disabling automerge for major updates explicitly?